subsys/bootloader/Kconfig

```diff
@@ -116,23 +116,43 @@ config SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST
 	  first. This is meant to be used for testing looping through the
 	  public keys.
 
-endif # SECURE_BOOT
-
-menuconfig IS_SECURE_BOOTLOADER
-	bool "Current app is bootloader"
-	depends on SECURE_BOOT_VALIDATION
-	depends on SECURE_BOOT_STORAGE
-	select SW_VECTOR_RELAY if SOC_SERIES_NRF51X
-	select NRFX_NVMC
+config SB_MONOTONIC_COUNTER
+	bool "Enable HW monotonic version counter"
+	default y
 	help
-	  This option is set by the first stage bootloader app to include all
-	  files and set all the options required.
+	  An array of slots is used for one counter. One slot is used per
```

Comment:
Can we have a sentence explaining what this is and what it is used for here?

Reply:
```suggestion
	  The counter stores the current firmware version in a way such that the value
	  can only increase. This is used to prevent malicious rollback of the firmware.
	  An array of slots is used for one counter. One slot is used per
```